curator-framework/src/main/java/org/apache/curator/framework/CuratorFrameworkFactory.java

```diff
@@ -401,6 +402,20 @@ public Builder zk34CompatibilityMode(boolean mode)
             return this;
         }
 
+        /**
+         * Set a default timeout for {@link CuratorZookeeperClient#close() }.
+         * The default is 0, which means that this feature is disabled.
+         *
+         * @since 4.0.2
+         * @param defaultWaitForShutdownTimeoutMs default timeout
+         * @return this
+         */
+        public Builder defaultWaitForShutdownTimeoutMs(int defaultWaitForShutdownTimeoutMs)
```

Comment:
Shouldn't this be just `waitForShutdownTimeoutMs()`? Normally, `CuratorZookeeperClient#close()` isn't called by users directly. Setting this value will cause the shutdown timeout to always be used.

Reply:
okay, done